src/Helper/Identity.php

```diff
@@ -4,57 +4,70 @@
 
 namespace Laminas\View\Helper;
 
-use Laminas\Authentication\AuthenticationServiceInterface;
-use Laminas\View\Exception;
+use function method_exists;
 
 /**
  * View helper plugin to fetch the authenticated identity.
  */
 class Identity extends AbstractHelper
 {
-    /**
-     * AuthenticationService instance
```

Comment:
IMO to be reverted: this code does not initialize an `AuthenticationSetvice`, but accepts an already existing one.

A type level declaration is preferable to going back to `object`, even if it leads to a soft dependency.

Stubbing it out could suffice, but the removal of it from `composer.json` potentially leads to conflicts with new major releases of the authentication component. Perhaps replacing `require` with `conflict` would suffice here.